### DISCUSSION TITLE
A place to document design decisions that have already been considered

### DISCUSSION THREAD

**Discussion by strangemonad** [modular/modular#206] [Ideas]
This seemed to serve the rust and Swift communities well. Github discussions are great but don't provide a way to summarize and organize multiple topics. 

I'm not imagining anything as involved as an RFC or Enhancement Proposal process but more of a Frequently Suggested Language Features with a few sentences on why the feature either would never make sense for mojo or didn't make sense at the time.

I'd think this is a place where the community could help consolidate documentation and discussions. Possibly an easy place to start with the open sourcing aspects might be with the documentation website while the source code and toolchain is still moving quickly?

**Comment by daydun** [comment]
https://mojodojo.dev/mojo_team_answers.html#language-features documents some stuff already, but more official documentation for justifications for and against language features would be great.

**Reply by strangemonad** [reply]
thanks @DayDun. Yep, I'm familiar with the awesome mojodev work. I was hoping for something more official that both modular staff and the community could contribute to

**Comment by lattner** [comment]
+1, this would be wonderful, both having this doc, and oss'ing docs (we just need to get more organized).  @scottamain
